### PR TITLE
Partner platform/component pull request

### PR DESCRIPTION
**Pull request by betzw**
Prepare handover to @bcostm

### CODE REVIEW

File: module.json [hunk 1]
```diff
@@ -20,11 +20,11 @@
   "dependencies": {
     "uvisor-lib": "^1.0.0",
     "cmsis-core": "*",
-    "mbed-hal-st-stm32f4": "^1.0.0"
+    "mbed-hal-st-stm32f4": "*"
```
Comment (0xc0170) [inline module.json]: why did you change it from ^ to * ?

Comment (betzw) [inline module.json]: This is a mistake. 
It comes from the period where I was still developing and wanted to get rid of version number warnings/conflicts.
Sorry about that.

Comment (betzw) [inline module.json]: Btw, is there a (semi-)automatic way to re-insert the actual dependencies into the module.json files?

File: module.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cmsis-core-stm32f4",
-  "version": "1.0.2",
+  "version": "1.1.0",
```
Comment (0xc0170) [inline module.json]: Can you revert this line change, otherwise I'll do it manually once merged

File: module.json (hunk 1 above)
Comment (0xc0170) [inline module.json]: Can you revert this changes - line 23 and line 27.

File: module.json [hunk 3]
```diff
@@ -20,11 +20,11 @@
   "dependencies": {
     "uvisor-lib": "^1.0.0",
     "cmsis-core": "*",
-    "mbed-hal-st-stm32f4": "^1.0.0"
+    "mbed-hal-st-stm32f4": "*"
   },
   "targetDependencies": {
     "stm32f401xe": {
-      "cmsis-core-stm32f401xe": "~0.2.0"
+      "cmsis-core-st-stm32f401xe": "*"
```
Comment (0xc0170) [inline module.json]: This was changed to follow other modules within cmsis. Shall we keep it cmsis-core-stm32f401xe?

File: module.json (hunk 1 above)
Comment (betzw) [inline module.json]: Can't you do it while mergeing?

File: module.json (hunk 2 above)
Comment (betzw) [inline module.json]: Pls. do it manually.

File: module.json (hunk 3 above)
Comment (betzw) [inline module.json]: Yes, please.
Remember to rename ARMmbed/cmsis-core-nucleo-f401re to ARMmbed/cmsis-core-st-stm32f401xe!
